tests/runtime/engine/runner.py

```diff
@@ -162,7 +187,16 @@ def run_test(test):
         after = None
         after_output = None
         cleanup = None
+        # This is only populated if the NEW_PIDNS directivce is set
+        # and is used to enter the newly created pid namespace for all BEFOREs,
+        # the prirmary RUN or PROG command, and the AFTER.
+        nsenter = []
         bpf_call = "[unknown]"
+
+        def get_pid_ns_cmd(cmd):
+            cwd = "." + os.getcwd() + "/"
+            return nsenter + [x.replace("./", cwd) for x in cmd.split()]
```

Comment:
Use abspath() like above? Also handles relative paths
```suggestion
            return nsenter + [os.path.abspath(x) for x in cmd.split()]
```